website/site/content/docs/widgetsV2.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Widgets V2
+position: 31
```

Comment:
Of course, we'll want to updated the title and filename and remove the old widgets.md before merging.